docs/videos/healthcare-and-life-sciences/Factor-Xa-example/submit.sh

```diff
@@ -32,33 +32,35 @@ grep -v -e HETATM -e CONECT ${PDB_FILE} >${PROTEIN}_protein.pdb
 mpirun -n 1 gmx_mpi pdb2gmx -f ${PROTEIN}_protein.pdb -o ${PROTEIN}_processed.gro -water tip3p -ff "charmm27"
 
 # Solvate System
-mpirun -n 1 gmx_mpi editconf -f ${PROTEIN}_processed.gro -o ${PROTEIN}_newbox.gro -c -d 1.0 -bt dodecahedron
+mpirun -n 1 gmx_mpi editconf -f ${PROTEIN}_processed.gro -o ${PROTEIN}_newbox.gro -c -d 1.75 -bt dodecahedron
 mpirun -n 1 gmx_mpi solvate -cp ${PROTEIN}_newbox.gro -cs spc216.gro -o ${PROTEIN}_solv.gro -p topol.top
 
-# Add ions
+# Add Ions
 mpirun -n 1 gmx_mpi grompp -maxwarn 1 -f config/ions.mdp -c ${PROTEIN}_solv.gro -p topol.top -o ions.tpr
 printf "SOL\n" | mpirun -n 1 gmx_mpi genion -s ions.tpr -o ${PROTEIN}_solv_ions.gro -conc 0.15 -p topol.top -pname NA -nname CL -neutral
 
-# Launch MPI jobs
-
 MDRUN_GPU_PARAMS="-gputasks 00 -bonded gpu -nb gpu -pme gpu -update gpu"
 MDRUN_MPIRUN_PREAMBLE="mpirun -n 1 -H localhost env GMX_ENABLE_DIRECT_GPU_COMM=1"
 
-# Run energy minimization
+# Run Energy Minimization
 mpirun -n 1 gmx_mpi grompp -maxwarn 1 -f config/emin-charmm.mdp -c ${PROTEIN}_solv_ions.gro -p topol.top -o em.tpr
 $MDRUN_MPIRUN_PREAMBLE gmx_mpi mdrun -v -deffnm em
 
-# Run temperature equilibration
+# Run Temperature Equilibration
 mpirun -n 1 gmx_mpi grompp -maxwarn 1 -f config/nvt-charmm.mdp -c em.gro -r em.gro -p topol.top -o nvt.tpr
 $MDRUN_MPIRUN_PREAMBLE gmx_mpi mdrun -v -deffnm nvt "$MDRUN_GPU_PARAMS"
```

Comment:
@vjb-collab says they were getting errors with the quotes on this line. Need to repro before we merge.

Reply:
I have updated to use shell array instead of string here.